vars.mk

```diff
@@ -11,7 +11,7 @@ KNORA_SIPI_REPO := knora-sipi
 KNORA_ASSETS_REPO := knora-assets
 KNORA_UPGRADE_REPO := knora-upgrade
 KNORA_SALSAH1_REPO := knora-salsah1
-KNORA_WEBAPI_DB_CONNECTIONS := 2
```

Comment:
Question: why there were two DB connections before?

Reply:
The idea was to try to limit the load on GraphDB in the hopes that the memory error will disappear. But it doesn't work. I will set it back to 2.